src/Helper/Request/LocaleHelper.php

```diff
@@ -36,7 +36,7 @@ public function redirectMissingLocale(Request $request): RedirectResponse
         } elseif (1 === \count($this->locales)) {
             $url = $request->getUriForPath('/'.$this->locales[0].$destination);
         } else {
-            $url = $this->router->generate('emsch_language_selection', ['destination' => $destination]);
+            $url = $request->getUriForPath('/'.$destination);
```

Comment:
Not sure about this, added todo before release (https://github.com/ems-project/website-skeleton/pull/45)

Reply:
On my side I would like to completely remove it. It can be handle in front end with local storage. Or with header sent by the browser. But I hate the bind locale (force a session on every request)

Reply:
Okay I take that in mind, and totally agree